server.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+
+require('dotenv').config();
+
+const express = require('express');
+const cors = require('cors');
+const climates = require('./data.weather.json');
+
+
+const PORT = process.env.PORT || 3001;
+
+const app = express();
+
+// middleware
+app.use(cors());
+
+
+app.get('/', (request, response) => {
+  response.status(200).send('I am home at last!');
+});
+
+app.get('/climates', (request, response) => {
```

Comment:
Hey there, I know you are still workin' away on this :)

Feedback here: see card 2 (make a get request to /weather) here